2021-01-Winter/0_modified_binary_search/README.md

```diff
@@ -28,63 +28,105 @@ In the style of:
 
 ### 1. PROBLEM 1 TODO :bug:
 
```

Comment:
Overall, awesome job! You included 3 questions too, which is above and beyond. 🥇